toolkit/tools/imager/imager.go

```diff
@@ -662,8 +662,10 @@ func configureDiskBootloader(systemConfig configuration.SystemConfig, installChr
 	// Add bootloader. Prefer a separate boot partition if one exists.
 	bootDevice, ok := installMap[bootMountPoint]
```

Comment:
not super important, but you could just rename `ok` -> `isBootSeparatePartition` here, would clarify the follow on conditional as well a bit.